content/posts/how-to-use-playwright-in-cucumberjs/index.md

```diff
@@ -0,0 +1,157 @@
++++
+date = ""
+author = "nploi"
+description = "How to use Playwright in cucumberjs"
+title = "How to use Playwright in cucumberjs"
+categories = ["DevSecOps", "Testing"]
+tags = ["bdd", "end-to-end", "cucumber", "automation", "test"]
+slug = "how-to-use-playwright-in-cucumberjs"
++++
+
```

Comment:
In our previous blog article [HERE](https://blog.manabie.io/2021/12/why-we-use-cucumber-for-end-to-end-testing/), we explained why we're using Cucumber for E2E test at Manabie, and provided a brief understanding about Cucumber.

This time, let's dive a little bit deeper!